src/make_table_example.F90

```diff
@@ -735,7 +820,7 @@ program make_table_example
      timestamp = dble(values(1))*10000.0d0+dble(values(2))*100.0+dble(values(3)) + &
           (dble(values(5))+dble(values(6))/60.0d0 + dble(values(7))/3600.0d0 )/24.0
 
-     if (doing_inelastic.or.doing_epannihil) then
+     if (doing_inelastic.or.doing_epannihil.or.doing_bremsstrahlung) then
```

Comment:
we should add in the `Itable_n_N` variable to the file name, like the res of the Itable variables